csm_web/scheduler/factories.py

```diff
@@ -16,6 +16,8 @@
     Override,
 )
 
+COMPSCI_WORDS = ('Algorithms', 'Systems', 'Distributed', 'Efficient', 'Tractable', 'Programming', 'Languages')
```

Comment:
why don't you throw in machine learning/blockchain while you're at it

Reply:
Will fix :)